golang/cosmos/app/app.go

```diff
@@ -851,6 +828,7 @@ func NewAgoricApp(
 			},
 			Deleted: []string{
 				crisistypes.ModuleName, // The SDK discontinued the crisis module in v0.51.0
+				"lien",                 // Agoric removed the lien module
```

Comment:
Lien storage *should* have never been used on any chain that we're upgrading, but it does no harm to properly clean up.